HISTORY.md

```diff
@@ -2,6 +2,7 @@ unreleased
 ==========
 
   * Set constructor name when possible
+  * deps: depd@1.1.2
```

Comment:
In case you missed my second comment (since it's hidden now):

> The other addition is that `deps:` will then list in a sub list all the changes that users would notice through the use of this module. There is one change in the `depd` package that they would in this case, the eval -> Function change. So add that as a sub bullet under the depd change 👍